open-company/open-company-web
# Company logo

card: https://trello.com/c/Y62sBxoC

test:
- open a company page
- click on the logo (should bring you to the company profile page)
- insert an invalid url in the logo field (it should fail also if you load a valid url that is not an image: http://asd.com for example)
- unfocus the field, can you see an error message? good
- insert a valid image url in the logo field (https://buffer.com/images/app/buffer-logo@2x.png for example)
- unfocus the field
- does the company logo in the top left corner of the nav bar change? good
- remove the url
- unfocus the field
- does the logo go back to the letter logo? good
- merge
